feature/bgp/admin_distance/README.md

```diff
@@ -0,0 +1,94 @@
+*   Create an IPv6 network i.e. ```ipv6-network-1 = 2024:db8:64:64::/64``` attached to ATE port-1 and port-2
+*   Configure IPv4 and IPv6 eBGP between DUT Port-1 and ATE Port-1
+    *   /network-instances/network-instance/protocols/protocol/bgp/global/config
+    *   /network-instances/network-instance/protocols/protocol/bgp/global/afi-safis/afi-safi/config/
+    *   Advertise ```ipv4-network-1 = 192.168.10.0/24``` and ```ipv6-network-1 = 2024:db8:64:64::/64``` from ATE to DUT over the IPv4 and IPv6 eBGP session on port-1
+*   Configure IPv4 and IPv6 IS-IS between DUT Port-2 and ATE Port-2
+    *   /network-instances/network-instance/protocols/protocol/isis/global/config
+    *   Advertise ```ipv4-network-1 = 192.168.10.0/24``` and ```ipv6-network-1 = 2024:db8:64:64::/64``` from ATE to DUT over the IPv4 and IPv6 IS-IS session on port-1
+
+### RT-1.34.1 [TODO:https://github.com/openconfig/featureprofiles/issues/3050]
+#### Validate traffic with default Route-Distance of eBGP
+*   Generate traffic from ATE port-3 towards ```ipv4-network-1 = 192.168.10.0/24``` and ```ipv6-network-1 = 2024:db8:64:64::/64```
+*   Verify that the traffic is received on port-1 of the ATE
+
+### RT-1.34.2 [TODO:https://github.com/openconfig/featureprofiles/issues/3050]
+#### Validate traffic with modified eBGP Route-Distance
+*   Configure Route-Distance of eBGP session on port-1 to 220
+    *   /network-instances/network-instance/protocols/protocol/bgp/global/default-route-distance/config/external-route-distance 
+*   Validate using gNMI Subscribe with mode 'ONCE' that the correct Route-Distance value of 220 is reported:
+    *   /network-instances/network-instance/protocols/protocol/bgp/global/default-route-distance/state/external-route-distance
+*   Generate traffic from ATE port-3 towards ```ipv4-network-1 = 192.168.10.0/24``` and ```ipv6-network-1 = 2024:db8:64:64::/64```
+*   Verify that the traffic is received on port-2 of the ATE
+
+### RT-1.34.3 [TODO:https://github.com/openconfig/featureprofiles/issues/3050]
+#### Validate traffic with default Route-Distance of iBGP
+*   Replace IPv4 and IPv6 eBGP with IPv4 and IPv6 iBGP between DUT Port-1 and ATE Port-1
+    *   /network-instances/network-instance/protocols/protocol/bgp/global/config
+    *   /network-instances/network-instance/protocols/protocol/bgp/global/afi-safis/afi-safi/config/
+    *   Advertise ```ipv4-network-1 = 192.168.10.0/24``` and ```ipv6-network-1 = 2024:db8:64:64::/64``` from ATE to DUT over the IPv4 and IPv6 iBGP session on port-1
+*   Generate traffic from ATE port-3 towards ```ipv4-network-1 = 192.168.10.0/24``` and ```ipv6-network-1 = 2024:db8:64:64::/64```
+*   Validate that the traffic is received on port-2 of the ATE
+
+### RT-1.34.3 [TODO:https://github.com/openconfig/featureprofiles/issues/3050]
+#### Validate traffic with modified iBGP Route-Distance
+*   Configure Route-Distance of iBGP session on port-1 to 40
```

Comment:
Same comment here, different implementations may use different IS-IS preferences (there are at least two implementations where IS-IS pref. will is in 10-20 range by default for internal IS-IS routes; I'm not sure how OTG device will advertise the prefix)

I'd suggest using the value <10 to ensure that iBGP is preferred.

Reply:
Probably should set both BGP [default-route-distance](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-protocols-protocol-bgp-global-default-route-distance) and ISIS [config/external-route-preference](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-protocols-protocol-isis-levels-level-route-preference-config-external-route-preference)

to ensure that BGP is preferred over ISIS

Reply:
That's an option, but I'm not sure that OTG encodes the is-is routes as external. It might be `internal-route-preference` instead (perhaps set both).

Reply:
@rohit-rp let's make the README be explicity and require the use of internal ISIS routes on OTG.  WHen the code is written we can find out if that is supported by OTG and update the README if necessary

Reply:
I am going to use BGP distance of 5 and 250 for the tests. I want to avoid configuring ISIS distance which is not the intent of the test. Distance of 5 and 250 will work for all vendors, ISIS falls between these values only. verified that OTG advertises as the networks as internal routes